src/bindeps_conda.jl

```diff
@@ -1,57 +1,46 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
 using BinDeps
 
-type Manager <: BinDeps.PackageManager
+type ManagerType{T} <: BinDeps.PackageManager
     packages::Vector{Compat.ASCIIString}
```

Comment:
Yes, it can be. In Conda.jl `Compat.UTF8String`, `Compat.ASCIIString`, `AbstractString` and `Compat.String` are all used. Which should be the standard?